setup.py

```diff
@@ -0,0 +1,41 @@
+from setuptools import setup
+
+package_name = 'rqt_msg'
+
+setup(
+    name=package_name,
+    version='1.0.2',
+    packages=[package_name],
+    package_dir={'': 'src'},
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name + '/resource',
+            ['resource/messages.ui']),
+        ('share/' + package_name, ['package.xml']),
+        ('share/' + package_name, ['plugin.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    author='Aaron Blasdel',
+    maintainer='Dirk Thomas, Dan Lazewatsky, Michael Lautman',
+    maintainer_email='dthomas@osrfoundation.org',
```

Comment:
Unrelated: Can we add the actual maintainers here?

I'm sure Dirk isn't active, no idea about the others.

Reply:
+1, but since we have to update the `package.xml` too, what about doing it in another PR? I think @mjcarroll is the maintainer now. I'm not sure.

Reply:
sure we can do it in another PR, it's unrelated to the purpose of this one